lab_stack/ConsoleApplication33/ConsoleApplication33/Stack.h

```diff
@@ -7,7 +7,7 @@ class Stack
     struct Element
     {
         int data;
-        Element* next; //��������� �� ��������� �������
+        Element* next;

```

Comment:
Init this as nullptr

Reply:
Well, i understood and corrected this mistake